### ISSUE TITLE
InvSee by request

### ISSUE THREAD

**Issue by SOF3** [BlockHorizons/InvSee#10]
```
when "$alice" runs "invseereq $bob"
    sendMessage("$bob", "$alice wants to see your inventory. See /invseegrant $alice to grant permission.")

when "$bob" runs "invseegrant $alice"
    "$alice" has permission to view "$bob"
```

I wanted to write this myself, but I can't bring myself into writing PocketMine plugins anymore. It is getting boring.

**Comment by lukeeey** [comment]
But admins should have the ability to see without granting

**Comment by SOF3** [comment]
Yes, I am not talking about admins.

The motivation is like this:

```
Alice: Drop your weapons by the door
Bob: *opens the chest by the door*
Alice: *sees only an iron sword in the chest*
Alice: I know you have a Sharpness V diamond sword. Put it down too!
Bob: But I left it at my base!
```

**Comment by Kris-Driv** [comment]
Who is Alice?

I could actually write this. My InvSee broke recently, so I was about to debug it myself anyways. I'll confirm this later message later on.

**Comment by SOF3** [comment]
Alice is the third party to Eve.
![](https://imgs.xkcd.com/comics/alice_and_bob.png)

**Comment by Kris-Driv** [comment]
I bet You waited for someone to ask you this 😆

**Comment by SOF3** [comment]
Actually I use Alice and Bob whenever I need to make up some names. They are the standard dummy player names in my test server too.
![](https://imgs.xkcd.com/comics/protocol_2x.png)

**Comment by UnknownOre** [comment]
lol

**Comment by Muqsit** [comment]
Implemented in 6875f1552c495f9273e9a540e50c51b1a1618c70 as a module
